apps/client/app/purchase-orders/page.tsx

```diff
@@ -1,8 +1,112 @@
+import Link from 'next/link';
+import moment from 'moment';
+import { sort } from 'fast-sort';
+import Currency from '../components/Currency';
+
+interface PurchaseOrderLineItem {
+  id: number;
+  item_id: number;
+  quantity: number;
+  unit_cost: number;
+  created_at: string;
+  updated_at: string;
+}
+
+interface PurchaseOrder {
+  id: number;
+  vendor_name: string;
+  order_date: string;
+  expected_delivery_date: string;
+  created_at: string;
+  updated_at: string;
+  purchase_order_line_items: PurchaseOrderLineItem[];
+}
+
+async function getPurchaseOrders(): Promise<PurchaseOrder[]> {
+  const res = await fetch('http://localhost:3100/api/purchase-orders', {
+    cache: 'no-cache',
+  });
+  if (!res.ok) {
+    throw new Error('Failed to fetch data');
+  }
+
+  const body = res.json();
+  return body;
+}
 
 export default async function Index() {
+  const purchaseOrders = await getPurchaseOrders();
+
+  const sortedPurchaseOrders = sort(purchaseOrders).asc(
```

Comment:
What is the default sort order of this data? Did you consider sorting in Prisma? What led you to this approach?